actors/common/src/main/java/org/sunbird/learner/actors/OrganisationManagementActor.java

```diff
@@ -1590,31 +1592,13 @@ private boolean validateUsrRequest(Map<String, Object> req) {
               JsonKey.ID,
               data.get(JsonKey.USER_ID));
     } else {
-      requestDbMap.put(JsonKey.PROVIDER, data.get(JsonKey.PROVIDER));
-      requestDbMap.put(JsonKey.USERNAME, data.get(JsonKey.USERNAME));
-      if (data.containsKey(JsonKey.PROVIDER)
-          && !StringUtils.isBlank((String) data.get(JsonKey.PROVIDER))) {
-        data.put(
-            JsonKey.LOGIN_ID,
-            (String) data.get(JsonKey.USERNAME) + "@" + (String) data.get(JsonKey.PROVIDER));
-      } else {
-        data.put(JsonKey.LOGIN_ID, data.get(JsonKey.USERNAME));
-      }
-
-      String loginId = "";
-      try {
-        loginId = encryptionService.encryptData((String) data.get(JsonKey.LOGIN_ID));
-      } catch (Exception e) {
-        ProjectCommonException exception =
-            new ProjectCommonException(
-                ResponseCode.userDataEncryptionError.getErrorCode(),
-                ResponseCode.userDataEncryptionError.getErrorMessage(),
-                ResponseCode.SERVER_ERROR.getResponseCode());
-        sender().tell(exception, self());
-      }
+      requestDbMap.put(JsonKey.USER_PROVIDER, data.get(JsonKey.USER_PROVIDER));
```

Comment:
put else if and check if external id available then - use this flow. else - old flow should continue